### PR TITLE
Demo to public

### PR DESCRIPTION
**Pull request by moluuser**
### Menu
* Hide Site Management
* InputHarnesses -> InputDemo
* Customers -> NestedFieldDemo
* Remove list editor example

### CMD
* Add reset DB cmd

### Login
* Add tips and account info to login page

### CODE REVIEW

File: example/cmd/data-restorer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM golang:alpine as builder
+RUN apk update && apk add git gcc libc-dev && rm -rf /var/cache/apk/*
+ARG GITHUB_TOKEN
```
Comment (idealhack) [inline example/cmd/data-restorer/Dockerfile:3]: Do we have any private dependency that needs this?
Also, it's not in the scope of the PR, but could you take a look at https://github.com/qor5/admin/blame/84962acce755363770fb247dd3d7de70024be66b/example/cmd/publisher/Dockerfile#L3 too?

File: example/cmd/data-restorer/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM golang:alpine as builder
```
Comment (idealhack) [inline example/cmd/data-restorer/Dockerfile:1]: Better to use another tag that has Go version pinned

File: example/cmd/data-restorer/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+RUN set -x && go get -d -v ./...
+RUN GOOS=linux GOARCH=amd64 go build -o /app/entry ./example/cmd/reset-data/
+
+FROM alpine:3.16
+RUN apk --update upgrade && \
+    apk add ca-certificates && \
+    apk add tzdata && \
+    rm -rf /var/cache/apk/*
+COPY --from=builder /app/entry  /bin/reset-data
+CMD /bin/reset-data
```
Comment (idealhack) [inline example/cmd/data-restorer/Dockerfile:15]: Both `restore` and `reset` are used, better to use the same name.

File: example/cmd/data-restorer/Dockerfile (hunk 1 above)
Comment (raven-chen) [reply example/cmd/data-restorer/Dockerfile:3]: No private dependency used here, I'll remove github token.(also in the other file)

**Comment by raven-chen** [comment]
Dup with https://github.com/qor5/admin/pull/57

**Review by raven-chen — APPROVED** [review]

**Pull request merged**